src/main/ruby/core/truffle/boot.rb

```diff
@@ -27,6 +27,8 @@ def self.check_syntax(source_or_file)
     1
   end
 
+  private_class_method :check_syntax
```

Comment:
When scanning our source code, having a space seems more frequent. I like the space better. Auto-formatter inserts the space as well and personally I have no desire to fight the auto-formatting. I would not mind `private_class_method def ...` pattern thought, but I think that's topic for another Rubocop rules adoption PR if there is a rule for it. I am inclined to keep the space.